lib/src/configuration.dart

```diff
@@ -67,9 +66,12 @@ class Configuration {
   /// The Apps name, defaults to [name] from pubspec
   late String name;
 
-  /// the Web Build folder, defaults to build/web
+  /// the Web Build folder, defaults to `web`
```

Comment:
Could also indicate that this is relative to build, if it indeed is.

Reply:
done